core/frontend/src/extension/ExtensionAdmin.ts

```diff
@@ -80,7 +80,7 @@ export class ExtensionAdmin {
       if (manifest.activationEvents.includes("onStartup"))
         provider.execute(); // eslint-disable-line @typescript-eslint/no-floating-promises
     } catch (e) {
-      throw new Error(`Failed to get manifest from extension: ${e}`);
+      throw new Error(`Failed to get extension manifest ${provider.hostname ? `at ${provider.hostname}` : ""}: ${(e as any).message}`);
```

Comment:
you shouldn't need to access the message property of the error....the error object itself should log sufficiently.....failing that, maybe type e: Error instead of casting to any